src/content/en/fundamentals/building-components/examples/howto-tooltip.md

```diff
@@ -84,8 +65,7 @@ limitations under the License.
 </div>
 
 
-<script src="https://cdn.rawgit.com/webcomponents/custom-elements/master/custom-elements.min.js"></script>
-<script src="https://cdn.rawgit.com/webcomponents/shadydom/master/shadydom.min.js"></script>
+<script src="https://cdn.rawgit.com/webcomponents/webcomponentsjs/d5b7ca65/webcomponents-sd-ce.js"></script>
```

Comment:
You can make the script error go away by indenting this script block by at least one space.